includes/class-wc-siftscience-element.php

```diff
@@ -0,0 +1,177 @@
+<?php
+/**
+ * This class creates ec element according to woocommearce library soecs.
+ *
+ * @author Nabeel Sulieman, Rami Jamleh
+ * @package sift-for-woocommerce
+ * @license GPL2
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+if ( ! class_exists( 'WC_SiftScience_Element' ) ) :
+
+	/**
+	 * Helper class for generating html
+	 *
+	 * Class WC_SiftScience_Element
+	 */
+	class WC_SiftScience_Element {
```

Comment:
How about we name this "WC_SiftScience_Input" or "WC_SiftScience_Input_Element" or "WC_SiftScience_Setting_Element". I think it is clearer

Reply:
not all of them are input elements input is understood as <input...   
settings is good however the prefix and suffix  WC_ _Element respectively do the job